minvws/nl-kat-coordination
# Clean more stale origins

### Changes

Clean up all non-inference type origins that have empty results in Octpoes/XTDB.

### Issue link

N/A

### Demo

N/A

### QA notes

Verify that, in a steady state, there are no non-inference type origins that have empty results in Octpoes/XTDB. This can be done using xtdb-cli.py or visually VisualOctopoesStudio.

To use xtdb-cli.py the following query checks for empty results for non-inference origins:
`octopoes/tools/xtdb-cli.py query '{:query {:find [(pull ?var [*])] :where [[?var :type "Origin"]]}}}' | jq -C '.[][] | select(.origin_type != "inference" and (.result | length == 0))'`

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
